aviary/interface/methods_for_level2.py

```diff
@@ -2516,12 +2516,12 @@ def save_sizing_to_json(self, json_filename='sizing_problem.json'):
 
                     # Lists are fine except if they contain enums
                     if type_value == list:
-                        if type(type(value[0])) == enum.EnumType:
+                        if type(type(value[0])) == type(enum.Enum):
```

Comment:
```suggestion
                        if isinstance(value[0], enum.Enum):
```
Should get you want you want but a little easier to read - `isinstance` will handle inheritance so you don't need to nest the `type()`s

Reply:
for some reason, isinstance was not triggering properly from some of the enums, but this did. I can experiment some more